docs/source/api_reference/utils.rst

```diff
@@ -293,3 +293,14 @@ Pytorch Utilities
 .. autofunction:: deepchem.utils.pytorch_utils.unsorted_segment_sum
 
 .. autofunction:: deepchem.utils.pytorch_utils.segment_sum
+
+Equivariance Utilities
+----------------------
```

Comment:
Add a paragraph here explaining these utilities and their general purpose to provide context

Reply:
Done.                                                                                                                    
Additionally, I would like to add a sentence like, you can check out the tutorials for more information about equivariance and Deepchem's equivariance support. However, I don't know how can I add hyperlinks in this section?

Reply:
Check out https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages